Classes/Player/PKPlayerSettings.swift

```diff
@@ -61,11 +62,25 @@ typealias SettingsChange = ((PlayerSettingsType) -> Void)
     /// In another case if you would like to start initializing the next media without buffering it, so that once the media is switched to the next one, it will be smother.
     @objc public var autoBuffer: Bool = true
     
+    @objc public var automaticallyPreservesTimeOffsetFromLive: Bool = false
```

Comment:
Move to dedicated class PKLowLatencyConfig or PKLowLatencySettings

Reply:
https://github.com/kaltura/playkit-android/blob/dev/playkit/src/main/java/com/kaltura/playkit/player/PKLowLatencyConfig.java